### PR TITLE
Show and hide poll results

### PR DESCRIPTION
**Pull request by MuhdNurKamal**
Add commands and GUI for show and hide poll results.
GUI fixes will be done after adding vote command

### CODE REVIEW

**Comment by MuhdNurKamal** [comment]
PR for task #139

File: src/main/java/seedu/club/ui/PollListPanel.java [hunk 1]
```diff
@@ -23,53 +22,62 @@
...
-                .addListener((observable, oldValue, newValue) -> {
-                    if (newValue != null) {
-                        logger.fine("Selection in poll list panel changed to : '" + newValue + "'");
-                        raise(new PollPanelSelectionChangedEvent(newValue));
-                    }
-                });
+    /**
+     * Shows results of polls
+     */
+    protected void showPollResults() {
```
Comment (yash-chowdhary) [inline src/main/java/seedu/club/ui/PollListPanel.java]: any reason why the access is `protected` ?

File: src/main/java/seedu/club/ui/TaskListPanel.java [hunk 2]
```diff
@@ -95,3 +95,6 @@ protected void updateItem(TaskCard task, boolean empty) {
         }
     }
 }
+
+
+
```
Comment (yash-chowdhary) [inline src/main/java/seedu/club/ui/TaskListPanel.java]: unnecessary spaces can be removed?

**Review by yash-chowdhary — COMMENTED** [review]
looks good overall!

File: src/main/java/seedu/club/ui/PollListPanel.java (hunk 1 above)
Comment (MuhdNurKamal) [reply src/main/java/seedu/club/ui/PollListPanel.java]: Ops. Initially thought was gonna call from out of class. Forget change to private. Thanks 😄

File: src/main/java/seedu/club/ui/TaskListPanel.java (hunk 2 above)
Comment (MuhdNurKamal) [reply src/main/java/seedu/club/ui/TaskListPanel.java]: Roger 👌

**Pull request merged**